coalib/results/TextPosition.py

```diff
@@ -32,6 +32,54 @@ def line(self):
     def column(self):
         return self._column
 
+    def __lt__(self, other):
+        """
+        Test whether ``self`` is behind the other
+        ``TextPosition``.
+
+        If the column in a ``TextPosition`` is ``None``, consider
+        whole line. If the line in a ``TextPosition`` is ``None``,
+        consider whole file.
+
+        :param other: ``TextPosition`` to compare with.
+        :return:      Whether this ``TextPosition`` is behind the other
+                      one.
+        """
+        if self.line is None or other.line is None:
+            return False
+        if self.line == other.line:
+            return (False
```

Comment:
@adtac 
Using regular if else may seem dummy here, as we'll be using something like
```Cython
if a:
  return True
else:
  if..
```
here if `a` is true, it'd return anyway. no need to put that in if else block.
Also, one more reason why I used separate if's is to clarify things when should it return true and when to return false according to None values, as discussed with makman2 and sils.